features/src/main/feature/camel-features.xml

```diff
@@ -1125,6 +1125,16 @@
         <bundle dependency='true'>mvn:org.ow2.asm/asm/9.3</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-jsonpath/${project.version}</bundle>
     </feature>
+    <feature name='camel-jta' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle dependency='true'>mvn:jakarta.transaction/jakarta.transaction-api/${jakarta-transaction-api-version}</bundle>
+        <bundle dependency='true'>mvn:jakarta.enterprise/jakarta.enterprise.cdi-api/${jakarta-enterprise-cdi-api-version}</bundle>
+        <bundle dependency='true'>mvn:jakarta.enterprise/jakarta.enterprise.lang-model/4.0.1</bundle>
```

Comment:
the value is the same as `jakarta-enterprise-cdi-api-version` parameter, but naming suggests that a separate parameter should be used

Reply:
there are newer versions, but they are either milestones or release candidates